obspy/io/nordic/tests/test_nordic.py

```diff
@@ -637,7 +637,7 @@ def test_large_negative_longitude(self):
             with warnings.catch_warnings():
                 warnings.simplefilter('ignore', UserWarning)
                 event_back = read_events(tf.name)
-        _assert_similarity(event, event_back[0])
+        _assert_similarity(event, event_back[0], verbose=True)
```

Comment:
This looks like it might have been commited by accident?

Reply:
Yup, thanks for catching it!